angular/bootstrap/src/components/select/select.component.ts

```diff
@@ -41,58 +41,24 @@ export class SelectItemDirective<Item> {
 	},
 	template: `
 		@if (state(); as state) {
-			<div
-				[auUse]="controlContainerDirective"
-				role="combobox"
-				class="d-flex align-items-center flex-wrap"
-				[attr.aria-controls]="state.id + '-menu'"
-				aria-haspopup="listbox"
-				[attr.aria-expanded]="state.open"
-			>
+			<div [auUse]="controlContainerDirective" class="d-flex align-items-center flex-wrap">
 				@if (state.selectedContexts; as selectedContexts) {
 					@for (itemContext of selectedContexts; track itemCtxTrackBy($index, itemContext)) {
-						<div
-							tabindex="-1"
-							class="au-select-badge me-1"
-							[class]="state.badgeClassName"
-							(keydown)="_widget.actions.onBadgeKeydown($event, itemContext.item)"
-						>
+						<div class="me-1" [auUse]="_widget.directives.badgeAttributesDirective" [auUseParams]="itemContext">
```

Comment:
I would propose to remove the `me-1` here and add a `gap-1` to the parent (the div where controlContainerDirective is applied).